components/brave_wallet/common/zcash_utils.h

```diff
@@ -17,12 +17,28 @@ namespace brave_wallet {
 
 inline constexpr size_t kZCashDigestSize = 32;
 inline constexpr size_t kOrchardRawBytesSize = 43;
+inline constexpr size_t kOrchardFullViewKeySize = 96;
 inline constexpr uint32_t kZip32Purpose = 32u;
 inline constexpr uint32_t kTestnetCoinType = 1u;
 inline constexpr uint32_t kDefaultZCashBlockHeightDelta = 20;
 inline constexpr uint32_t kDefaultTransparentOutputsCount = 2;
 inline constexpr uint32_t kGraceActionsCount = 2;
 inline constexpr uint64_t kMarginalFee = 5000;
+// Parts of compact orchard compact action
+// https://github.com/zcash/lightwalletd/blob/6e3816b5834583c492c37ce05b0faaf9fe12c87f/walletrpc/compact_formats.proto#L75
+inline constexpr size_t kOrchardNullifierSize = 32u;
+inline constexpr size_t kOrchardCmxSize = 32u;
+inline constexpr size_t kOrchardEphemeralKeySize = 32u;
+inline constexpr size_t kOrchardCipherTextSize = 52u;
+
+using OrchardFullViewKey = std::array<uint8_t, kOrchardFullViewKeySize>;
+
+// Reduce current scanning position on this value if reorg is found
+// All Zcash network participants basically assume rollbacks longer than 100
+// blocks will not occur
+inline constexpr size_t kChainReorgBlockDelta = 150;
+// Number of blocks downloaded by a single request
+inline constexpr size_t kScanBatchSize = 10;
```

Comment:
nit: Chromium style guide https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar recommends using complete sentences as comments so in most cases they should end with a period `.`